### DISCUSSION TITLE
How to identify which NPM packages are supported on vscode.dev?

### DISCUSSION THREAD

**Discussion by amitjoshi438** [microsoft/vscode-discussions#1036] [Extension Development QnA] [answered]
How to identify which NPM packages are supported on vscode.dev? I understand that we don't have server support on vscode.dev but not sure what should I look for when deciding which dependency will work on vscode.dev.

**Comment by isidorn** [comment, marked as answer]
NPM modules in general will not run on vscode.dev
This docs have more details https://code.visualstudio.com/api/extension-guides/web-extensions

**Comment by abrilurena** [comment]
Hi, @amitjoshi438! It's been a while, I just wanted to check in and see if you still need any help with this?

**Comment by amitjoshi438** [comment]
Thanks for reminding @abrilurena, I am working on a web extension and used following [NPM package] (https://www.npmjs.com/package/xmldom) and it was working on vscode.dev. So, bit confused around using NPM packages on vscode.dev.
